apps/ukvi-complaints/translations/src/en/fields.json

```diff
@@ -190,6 +199,30 @@
       "notUrl": "Enter the Unique Application Number"
     }
   },
+  "case-id-reference": {
+    "label": "Enter your Case ID number",
+    "hint": null,
+    "validation": {
+      "required": "Enter your Case ID number",
+      "notUrl": "Enter your Case ID number"
```

Comment:
if the user enters a url and they're not allowed to they should receive a more specific validation message 
have a look at brp service for the wording